### PR TITLE
Add helm chart for cloudwatch

### PR DESCRIPTION
**Pull request by bnrjee**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: helm-charts/stable/cloudwatch-agent/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
```
Comment (abhinavmpandey08) [inline helm-charts/stable/cloudwatch-agent/Chart.yaml:17]: Can you set the initial version and app version to 1.0.0 to match our other charts?
https://github.com/aws/eks-distro-build-tooling/blob/main/helm-charts/stable/prow-data-plane/Chart.yaml#L32-L37
https://github.com/aws/eks-distro-build-tooling/blob/main/helm-charts/stable/prow-control-plane/Chart.yaml#L32-L37

File: helm-charts/stable/cloudwatch-agent/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: 1.16.0
```
Comment (abhinavmpandey08) [inline helm-charts/stable/cloudwatch-agent/Chart.yaml:22]: Can you set the initial version and app version to 1.0.0 to match our other charts?
https://github.com/aws/eks-distro-build-tooling/blob/main/helm-charts/stable/prow-data-plane/Chart.yaml#L32-L37
https://github.com/aws/eks-distro-build-tooling/blob/main/helm-charts/stable/prow-control-plane/Chart.yaml#L32-L37

**Comment by abhinavmpandey08** [comment]
Can you also add the following apache license to all yamls in templates/ and Chart.yaml and values.yaml
https://github.com/aws/eks-distro-build-tooling/blob/main/helm-charts/stable/prow-control-plane/Chart.yaml#L1-L13

**Comment by TerryHowe** [comment]
Is this good to go or outdated?

**Review by micahhausler — CHANGES_REQUESTED** [review]
Thanks for this! one change requested

**Comment by bnrjee** [comment]
These changes will be tracked with 
https://github.com/aws/eks-distro-build-tooling/pull/82

**Pull request closed without merging**